### ISSUE TITLE
Feature Request: Mechanism for Code Highlighting

### ISSUE THREAD

**Issue by jpmedley** [mdn/content#1761]
I've seen a number off PRs removing manual code highlighting. (ex: #1372) Clearly such highlighting will go away when the content is converted to markdown. Removing such highlighting now overlooks the reason such highlighting may have been added in the first place; to draw readers attention to important details. 

I don't know exactly how you would do such a thing in HTML. For a reference implementation, here's what we do on web.dev. https://web.dev/handbook/markup-code/#code-highlighting

**Comment by Ryuno-Ki** [comment]
Related to https://github.com/mdn/yari/issues/2198

**Comment by hamishwillee** [comment]
P.S. Note the last [comment I made in that issue](https://github.com/mdn/yari/issues/2198#issuecomment-769489840) which basically says we would like highlighting in markdown too.

**Comment by jpmedley** [comment]
I agree. Look to the future. What I'm objecting to is abandoning the past before the replacement is ready to use.

**Comment by hamishwillee** [comment]
Yes. Blanket removal of this markup without looking at why it was added is wrong. That said, I have given up hope of a supported HTML method appearing so have moved on to chasing this for markdown.
